octodns/provider/route53.py

```diff
@@ -211,6 +468,35 @@ def __repr__(self):
                                                           self.values)
 
 
+_mod_keyer_action_order = {
+    'DELETE': '0',  # Delete things first
+    'CREATE': '1',  # Then Create things
+    'UPSERT': '2',  # Upsert things last
+}
+
+
+def _mod_keyer(mod):
+    rrset = mod['ResourceRecordSet']
+    action_order = _mod_keyer_action_order[mod['Action']]
+
+    # We're sorting by 3 "columns", the action, the rrset type, and finally the
+    # name/id of the rrset
+
+    if rrset.get('GeoLocation', False):
+        return '{}3{}'.format(action_order, rrset['SetIdentifier'])
```

Comment:
Any reason we can't return a tuple to sort by, e.g. `(action_order, 3, rrset['SetIdentifier'])`? I think that would have the same sorting properties but avoid needing to munge it into a string.

Reply:
No, just didn't occur to me when things were evolving through various ordering options. Pushing up a switch shortly.